### DISCUSSION TITLE
cannot find program: host

### DISCUSSION THREAD

**Discussion by Fanto1978** [matteocorti/check_ssl_cert#409] [General]
i launched check_ssl-cert -H www.webserveracaso.it 

SSL_CERT UNKNOWN www.webserveracaso.it : cannot find program: host

what's wrong?

**Comment by matteocorti** [comment]
It appears that you do not have the ```host``` utility. Which system are you using? Which is the output of ```command -v host```?

**Reply by Fanto1978** [reply]
oh my god... mea culpa :D

installed bind-utils and nmap, now it works fine.... sorry ...

i'm using almalinux with centreon mon-system (nagios fork)

thank you very much...
